Echo.Core/src/Textures/Colors/RGBA128.Parser.cs

```diff
@@ -171,56 +165,72 @@ bool ParseRGB(out RGBA128 result)
 			return YieldError(out result);
 
 			static bool IsChannelInRange(int channel) => channel > 0 && channel <= 255;
-			static bool findNextInt(ReadOnlySpan<char> src, ref int index, ref int result)
+			static bool FindNextInt(ReadOnlySpan<char> source, ref int head, out int result, bool alpha = false)
 			{
-				int len = 0;
-
-				while (!char.IsDigit(src[index])) if (src.Length <= index++) return false; // find first digit
-				while (char.IsDigit(src[index + len]) && (index + len) < (src.Length - 1)) len++; // get the length of the number
+				if (alpha)
+				{
+					result = 255;
+					return true;
+				}

```

Comment:
Should not return this early; this would ignore the alpha channel if we actually have one.